xbmc/platform/android/PlatformAndroid.cpp

```diff
@@ -34,3 +37,17 @@ bool CPlatformAndroid::Init()
 
   return true;
 }
+
+void CPlatformAndroid::PlatformSyslog()
+{
+  CLog::Log(
+      LOGINFO,
+      "Product: %s, Device: %s, Board: %s - Manufacturer: %s, Brand: %s, Model: %s, Hardware: %s",
+      CJNIBuild::PRODUCT.c_str(), CJNIBuild::DEVICE.c_str(), CJNIBuild::BOARD.c_str(),
+      CJNIBuild::MANUFACTURER.c_str(), CJNIBuild::BRAND.c_str(), CJNIBuild::MODEL.c_str(),
+      CJNIBuild::HARDWARE.c_str());
+  std::string extstorage;
+  bool extready = CXBMCApp::GetExternalStorage(extstorage);
+  CLog::Log(LOGINFO, "External storage path = %s; status = %s", extstorage.c_str(),
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)